tests/models/test_ftrl.py

```diff
@@ -181,6 +190,15 @@ def test_ftrl_construct_wrong_nbins_value():
             == str(e.value))
 
 
+@pytest.mark.parametrize('value, message',
+                         [[-1, "Argument `mantissa_nbits` in Ftrl() constructor cannot be negative: -1"],
+                         [54, "Argument `mantissa_nbits` in Ftrl() constructor should be less than 54, got: 54"]])
```

Comment:
mantissa is only 52 bits in doubles

Reply:
Yes, typo when going from `float_nbits` to `mantissa_nbits`, let me fix it.